tests/unit/test_reconcile.py

```diff
@@ -92,6 +92,40 @@ def test__local_and_server_error(
         )
 
 
+def test__get_server_content_missing_link():
+    """
+    arrange: given path info with a file and table row with no changes and discourse client that
+        raises an error
+    act: when _get_server_content is called with the path info and table row
+    assert: then ServerError is raised.
```

Comment:
`ReconcilliationError` and not `ServerError`

Reply:
Thanks for spotting this, should be fixed shortly